acord/models/guild.py

```diff
@@ -103,7 +111,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     nsfw: bool
     """ Whether the guild is marked as NSFW """
-    nsfw_level: int
+    nsfw_level: NSFWLevel
```

Comment:
:class:\`NSFWLevel\`

Reply:
Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c